public/js/fortune.js

```diff
@@ -1,8 +1,8 @@
-// update the random fortune page using AJAX
+/// update the random fortune page using AJAX
 //
 $("#random-fortune").on("click", function(event) {
   event.preventDefault();
-  $.get("/api/v1/fortunes/random.json", function(newFortune) {
+  $.get("/fortunes/random.json", function(newFortune) {
```

Comment:
We shouldn't change the path, here. I was making an example of having the form post to one path, and AJAX posting to another.